### PR TITLE
[2.9] UI guide updates

### PR DESCRIPTION
**Pull request by TJaniF**
New screenshots and gifs mostly :)

### CODE REVIEW

**Review by kentdanas — APPROVED** [review]
Looks great! Good to merge once we've confirmed nothing else will change in the UI in teh final release

**Comment by TJaniF** [comment]
@jwitz and @lzdanski This one mostly screenshot updates, just very few copy updates to go along with them :)

File: learn/airflow-ui.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ One of the main features of Airflow is its [user interface (UI)](https://airflow
 
 This guide is an overview of some of the most useful features and visualizations in the Airflow UI. If you're not already using Airflow and want to get it up and running to follow along, see [Install the Astro CLI](https://docs.astronomer.io/astro/cli/get-started) to quickly run Airflow locally.
 
-All images in this guide were taken from an [Astronomer Runtime](https://docs.astronomer.io/astro/runtime-release-notes) Airflow image. Other than some modified colors and an additional **Astronomer** tab, the UI is the same as when using OSS Airflow. The images in this guide are from Airflow version 2.7, if you are using an older version of Airflow, some UI elements might be slightly different or missing.
+All images in this guide were taken from an [Astronomer Runtime](https://docs.astronomer.io/astro/runtime-release-notes) Airflow image. Other than some modified colors and an additional **Astronomer** tab, the UI is the same as when using OSS Airflow. The images in this guide are from Airflow version 2.9, if you are using an older version of Airflow, some UI elements might be slightly different or missing.
```
Comment (lzdanski) [inline learn/airflow-ui.md:12]:
```suggestion
All images in this guide were taken from an [Astronomer Runtime](https://docs.astronomer.io/astro/runtime-release-notes) Airflow image. Other than some modified colors and an additional **Astronomer** tab, the UI is the same as when using OSS Airflow. The images in this guide are from Airflow version 2.9. If you are using an older version of Airflow, some UI elements might be slightly different or missing.
```

**Review by lzdanski — APPROVED** [review]
One small grammar change! Checked it out in preview, LGTM

**Review by TJaniF — COMMENTED** [review]
Thank you @lzdanski :)

**Pull request merged**